video_creation/voices.py

```diff
@@ -25,7 +26,9 @@ def save_text_to_mp3(reddit_obj):
         # ! Stop creating mp3 files if the length is greater than 50 seconds. This can be longer, but this is just a good starting point
         if length > 50:
             break
-        tts = gTTS(text=comment["comment_body"], lang="en", slow=False)
+        comment=comment["comment_body"]
+        text=re.sub('((http|https)\:\/\/)?[a-zA-Z0-9\.\/\?\:@\-_=#]+\.([a-zA-Z]){2,6}([a-zA-Z0-9\.\&\/\?\:@\-_=#])*', '', comment)
```

Comment:
Correct me if I'm wrong but that regex expression looks overly complicated for something that could be as simple as `((http|https)://[^\s]+)`

[Demo](https://regex101.com/r/cuM7e2/1)

Reply:
That would work most of the time, but in cases where the link is just `www.example.com`, or even just `example.com`, It would not. That's why it seems complicated because it covers a lot of edge cases.
You can use [RegExr](https://regexr.com/) to explore the diffrences.